includes/class-kirki-field.php

```diff
@@ -537,8 +537,8 @@ protected function set_tooltip() {
 		 */
 		protected function set_active_callback() {
 
-			if ( is_array( $this->active_callback ) && ! is_callable( $this->active_callback ) ) {
-				if ( isset( $this->active_callback[0] ) && is_array( $this->active_callback[0] ) ) {
+			if ( is_array( $this->active_callback ) ) {
+				if ( isset( $this->active_callback[0] ) ) {
 					$this->required = $this->active_callback;
```

Comment:
@aristath You are right, the change at line 540 is unnecessary
The change at line 541 on the other hand should be kept to allow functions in the global scope to be called